modules/rosapps/applications/sysutils/utils/pice/module/utils.c

```diff
@@ -803,7 +803,7 @@ ULONG GetLinearAddress(USHORT Segment,ULONG Offset)
 		__asm__("\n\t \
 			sldt %%ax\n\t \
 			mov %%ax,%0"
-			:"=m" (Segment));
+			:"=m" (Segment) : : "ax");
```

Comment:
I checked the [latest source code](https://sourceforge.net/projects/pice/files/pICE%20source/), and didn't found this change. I think it should be upstreamed.

If the project is not maintained at upstream, you should use this construct:
```c
#ifdef __REACTOS__
  // new code
#else
  // old upstream code
#endif
```

Reply:
Very thanks for your remind, I'm mining some misuses of inline assembly on github and found the above misuse in this project. I was not realized that this needs to be submitted to upstream project, thanks very much.